build.gradle

```diff
@@ -316,7 +318,9 @@ project('spring-integration-core') {
 		}
 		compile("io.fastjson:boon:$boonVersion", optional)
 		compile("com.esotericsoftware:kryo-shaded:$kryoShadedVersion", optional)
+		compile "io.micrometer:micrometer-core:$micrometerVersion"
 
+		testCompile "org.assertj:assertj-core:$assertjVersion"
```

Comment:
Since we agreed about supporting AssertJ this has to go to the `spring-integration-test-support` to let all the modules to pull transitive AssertJ for free.